webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -43,11 +65,64 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
 
   const [canceler] = useState(new AbortController());
   const [models, setModels] = useState<Loadable<ModelItem[]>>(NotLoaded);
+  const [taskAllocation, setTaskAllocation] =
+    useState<Loadable<AllocationData | undefined>>(NotLoaded);
   const [selectedModelName, setSelectedModelName] = useState<string>();
+  const shouldRenderAllocationCard = useMemo(
```

Comment:
do you need usememo here?

Reply:
does that cause many re-render without useMemo?

Reply:
what do you mean by standard?
if there's no good reason, i think its good to remove it.

Reply:
do we need this useMemo?